TESTS/netsocket/README.md

```diff
@@ -18,6 +18,7 @@ The target for this plan is to test:
 -   [UDPSocket](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/UDPSocket.h).
 -   [TCPSocket](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/TCPSocket.h).
 -   [TLSSocket](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/TLSSocket.h).
+-   [DNS](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/DNS.h)
 
 Reference documentation: https://os.mbed.com/docs/latest/reference/network-socket.html
```

Comment:
OK, not part of your changes but I guess this should be changed to https://os.mbed.com/docs/mbed-os/latest/apis/network-socket.html . And maybe also convert it to proper link named as "Network Socket Documentation" - or something similar